src/main/java/frc/robot/poseestimator/limelights/LimelightRawData.java

```diff
@@ -0,0 +1,5 @@
+package frc.robot.poseestimator.limelights;
+
+import edu.wpi.first.math.geometry.Pose2d;
+
+public record LimelightRawData(Pose2d estimatedPosition, double aprilTagHeight, double distanceFromAprilTag, double timeStamp) {}
```

Comment:
maybe find a way to use EntryValue instead of getting only 2 of them

Reply:
The only way is to pass a entry to the record, or a consumer that returns supplier that does those kind of things. I don't like that, either way.